src/main/scala/org/globalforestwatch/layers/ArgOTBN.scala

```diff
@@ -0,0 +1,21 @@
+package org.globalforestwatch.layers
+
+import org.globalforestwatch.grids.GridTile
+
+case class ArgOTBN(gridTile: GridTile, kwargs: Map[String, Any])
+    extends StringLayer
+        with OptionalILayer {
+
+    val datasetName = "arg_native_forest_land_plan"
+    val uri: String = 
+        uriForGrid(gridTile)
```

Comment:
This function takes kwargs argument now. I had to merge `develop` into my PRODES [PR](https://github.com/wri/gfw_forest_loss_geotrellis/pull/171), make that update and make sure tests pass.

Reply:
fixed in [latest commit](https://github.com/wri/gfw_forest_loss_geotrellis/pull/172/commits/6492a97357531c7c8cb97011073b4cc486006b74)